src/main/java/com/thealgorithms/maths/Area.java

```diff
@@ -192,4 +192,24 @@ public static double surfaceAreaCone(final double radius, final double height) {
         }
         return Math.PI * radius * (radius + Math.pow(height * height + radius * radius, 0.5));
     }
+
+
+    /**
+     * Calculate the surface area of a pyramid with a square base.
+     *
+     * @param sideLength side length of the square base
+     * @param slantHeight slant height of the pyramid
+     * @return surface area of the given pyramid
+     */
+    public static double surfaceAreaPyramid(final double sideLength, final double slantHeight) {
+        if (sideLength <= 0) {
+            throw new IllegalArgumentException("Must be a positive sideLength");
+        }
+        if (slantHeight <= 0) {
+            throw new IllegalArgumentException("Must be a positive slantHeight");
```

Comment:
Can we also check for not null?

Reply:
@anandfresh Actually we... can't. `double` is a _primitive type_ so that cannot be `null`. There is even no operator to compare `double` with `null`.